### ISSUE TITLE
talkback does not read editbox string V3.3.1

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#233]
```
What steps will reproduce the problem?
1. talkback on
2. calculator input some text
3. remove number (expect read the remain number.)

What is the expected output? What do you see instead?
Device should read the remain number.

What version of the product are you using? On what operating system?
JB 4.2 talkback 3.3.1

Please provide any additional information below.
In talback 3.2.1, working properly.
This issue occurs in Talkback version 3.3.1.
```

Original issue reported on code.google.com by `ezco...@gmail.com` on 22 Apr 2013 at 1:49

**Comment by GoogleCodeExporter** [comment]
```
04-22 01:44:56.352  8043  8043 D ProcessorEventQueue: Processing event: 
EventType: TYPE_VIEW_CLICKED; EventTime: 3222644; PackageName: 
com.android.calculator2; MovementGranularity: 0; Action: 0 [ ClassName: 
android.widget.ImageButton; Text: [Delete]; ContentDescription: Delete; 
ItemCount: -1; CurrentItemIndex: -1; IsEnabled: true; IsPassword: false; 
IsChecked: false; IsFullScreen: false; Scrollable: false; BeforeText: null; 
FromIndex: -1; ToIndex: -1; ScrollX: -1; ScrollY: -1; MaxScrollX: -1; 
MaxScrollY: -1; AddedCount: -1; RemovedCount: -1; ParcelableData: null ]; 
recordCount: 0
04-22 01:44:56.376  8043  8043 V EventSpeechRuleProcessor: Processed event 
using rule:
04-22 01:44:56.376  8043  8043 V EventSpeechRuleProcessor: <ss:rule>
04-22 01:44:56.376  8043  8043 V EventSpeechRuleProcessor:     <ss:filter>
04-22 01:44:56.376  8043  8043 V EventSpeechRuleProcessor:       
<ss:eventType>TYPE_VIEW_CLICKED</ss:eventType>
04-22 01:44:56.376  8043  8043 V EventSpeechRuleProcessor:     </ss:filter>
04-22 01:44:56.376  8043  8043 V EventSpeechRuleProcessor:     <ss:metadata>
04-22 01:44:56.376  8043  8043 V EventSpeechRuleProcessor:       
<ss:customVibration>@id/patterns_clicked</ss:customVibration>
04-22 01:44:56.376  8043  8043 V EventSpeechRuleProcessor:       
<ss:customEarcon>@id/sounds_clicked</ss:customEarcon>
04-22 01:44:56.376  8043  8043 V EventSpeechRuleProcessor:     </ss:metadata>
04-22 01:44:56.376  8043  8043 V EventSpeechRuleProcessor:   </ss:rule>
04-22 01:44:56.383  8043  8043 D ProcessorEventQueue: Processing event: 
EventType: TYPE_VIEW_FOCUSED; EventTime: 3222655; PackageName: 
com.android.calculator2; MovementGranularity: 0; Action: 0 [ ClassName: 
android.widget.EditText; Text: [55555]; ContentDescription: null; ItemCount: 
20; CurrentItemIndex: 1; IsEnabled: true; IsPassword: false; IsChecked: false; 
IsFullScreen: false; Scrollable: false; BeforeText: null; FromIndex: -1; 
ToIndex: -1; ScrollX: -1; ScrollY: -1; MaxScrollX: -1; MaxScrollY: -1; 
AddedCount: -1; RemovedCount: -1; ParcelableData: null ]; recordCount: 0
04-22 01:44:56.407  8043  8043 V EventSpeechRuleProcessor: The "DefaultFilter" 
filter accepted the event, but the "FallbackFormatter" formatter indicated the 
event should be dropped.
04-22 01:44:56.415  8043  8043 W ProcessorEventQueue: Failed to process event

```

Original comment by `ezco...@gmail.com` on 22 Apr 2013 at 1:51

**Comment by GoogleCodeExporter** [comment]
```
Issue with the Calculator app. Bug filed against calculator. Cannot fix from 
TalkBack without introducing regressions in general text editing experience.
```

Original comment by `alanv@google.com` on 23 Apr 2013 at 2:43
- Changed state: **Invalid**
